examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -5,11 +5,22 @@ title: Robust Lab Example
 description: Seriously the best lab you've ever taken. Bar none.
 default_locale: en
 duration: 60
-level: easy
+max_duration: 60
+credits: 5
+level: fundamental
+logo: source-code.png
 tags:
 - sample
 - life-changing
 - gcp
+product_tags: ['app engine', 'kubernetes engine', 'cloud functions']
+role_tags: ['cloud architect', 'developers backend', 'developers frontend']
+domain_tags:
+- infrastructure
+- application development
+- data
```

Comment:
Are product_tags, role_tags, domain_tags ever used? I remember failing to find an example. Maybe @robsideshow knows. If not, we should not add them to the example.

Reply:
- Recently product_tags, role_tags, domain_tags are not used. you can check that [here](https://github.com/CloudVLab/gcp-brex/blob/b22611ca59b5ede434a79b50d419158b4901e5be/labs/qt_oct23_week3/qwiklabs.es.yaml#L10).
This is cl for adding this attribute to [g3](https://critique.corp.google.com/cl/324780113/depot/google3/cloud/training/qwiklabs/content_bundle/examples/lab-v1-robust/qwiklabs.yaml)
- For legacy_display_options last time we used this attribute from [4 years ](https://github.com/CloudVLab/qwiklabs-test-content/blob/ff5271ea63f5fa1a8b943d4c87e8ab32690ad861/labs/test_v1_lab_migration_5/qwiklabs.yaml#L10)
this [Cl](https://critique.corp.google.com/cl/258640416/depot/google3/cloud/training/qwiklabs/content_bundle/schema/content_bundle_lab.yaml) for that

Reply:
If they're not used, I think we should not add them to the example